solr/solr-ref-guide/modules/configuration-guide/pages/cluster-plugins.adoc

```diff
@@ -88,24 +88,22 @@ A JSON map of additional plugin configuration parameters.
 Plugins that implement `ConfigurablePlugin` interface will be initialized with a
 plugin-specific configuration object deserialized from this map.
 
-Example plugin configuration:
+Example plugin with configuration:
 
-[source,bash]
+[source,json]
 ----
-curl -X POST -H 'Content-type: application/json' -d '{
```

Comment:
I did it because I was trying to fit in the same page documentation of plugins configured at node- and cluster-level. This command was shown when describing what a plugin looks like in JSON, before the cluster API was even presented. What matters here is hence the contents of the JSON, the curl command appeared as noise to me - and a forward reference to some concept not already presented.

The way I was seeing it, documentation should present three things:
1. A general introduction of node plugins, and what their properties are.
2. How to configure them in solr.xml
3. How to configure them with cluster API